packages/ui/src/message-room-details-dialog/MessageRoomDetailsDialog.jsx

```diff
@@ -252,8 +252,40 @@ function MembersPanel({ config }) {
     addmembersevent(values);
   };
 
+  //to remove member from the channel
+  //the id was passed from the RemoveMemberModal component to be able to identify the id we want to remove
+  // when we destructure the member we want to remove
+  //this  removememberevent(id); is just to console that the plugin/member has being delete
   const removeMemberEvent = id => {
+    //to get the organization id from the local storage
+
+    const theUserData = JSON.parse(localStorage.getItem("userData"));
+
+    console.log(theUserData.user.org_id);
+
+    const theOrganizarionId = theUserData.user.org_id;
+
+    //to get the current room , which we have in the session storage
+
+    let ourCurrentRoom = sessionStorage.getItem("currentRoom");
+
+    const token = sessionStorage.getItem("token");
+
+    axios
+      .delete(
+        `http://chat.zuri.chat/api/v1/org/${theOrganizarionId}/rooms/${ourCurrentRoom}/members/${id}`,
```

Comment:
This request will delete a user from the organisation instead of removing them from a room
Check the api's on https://chat.zuri.chat/docs to find the PATCH request that can be used to remove a member from a channel